src/core.js

```diff
@@ -620,9 +624,13 @@ $.extend( $.validator, {
 			this.toHide = this.errors().add( this.containers );
 		},
 
+		setToHideFor: function( element ) {
```

Comment:
I am fine with not adding this new method but leave the 2 call sites (with a duplicated line)